content/faq/42_cost-to-participate-in-the-challenge/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: how much would it cost to participate in the challenge?
+type: "faq"
+priority: "42"
+tags: ["participation", "registration"]
+---
+
+Short answer to that question is, participation won't cost you anything. You don't need to spend a single dime in order to participate.
+
+All you have to do is to send a PR to the Github repo. If your PR gets in, you'll be registered.\
+That's all. You don't need to pay any kind of registration fees either.\
+If someone tries to tell you otherwise, or misguide you, please feel free to report this issue to us in our social media handles immediately. We'll gladly help you.
+
+However, me and our team of mentors are going to suggest you some courses that'll help you to learn things in order to win the challenge. Most of these courses will be free, but we'll also share some paid courses from reputed learning platforms like Udemy and Coursera. These courses won't cost you more than 500-600 Rupees, that is way less amount than what you pay your college or any other educational institution.
```

Comment:
INR

Reply:
instead of rupees say INR.
Also, take care of time and currency internationalisation at other places

Reply:
Noted!